app/controllers/frontend_log_controller.rb

```diff
@@ -48,7 +48,7 @@ def log_params
   end
 
   def check_user_authenticated
-    return if effective_user
+    return if effective_user || request.base_url
```

Comment:
Is this change related to tracking the click events?

Reply:
Yes, it was. I was trying to figure out a way to track the event on both the home screen and the "Choose a password" screen. With this, I was able to trigger the analytics event on the home page and if this is a sufficient way that I could potentially target that choose a password page.

Reply:
Ah, I see. So this was preventing access because we're requiring the user to be signed in, which won't be the case for the sign-in screen.

Should we just remove this check altogether? Admittedly I don't have the context if there was a strong reason for why we wanted the user to be authenticated for frontend event logging, but I think we need to make a choice to either remove the authentication check altogether and allow unauthenticated logging, or keep the authentication requirement and revisit the requirements of the ticket.

Reply:
Yes because if you aren't logged in, it would throw a 401 error. I can bring it up for discussion and see who knows or has that context. Then from there we can either remove it, or revisit requirements.

Reply:
@jmdembe Can you summarize where we landed with this? I see the `before_action` has been removed, but I'm not sure how we arrived at that decision.

Reply:
> @jmdembe Can you summarize where we landed with this? I see the `before_action` has been removed, but I'm not sure how we arrived at that decision.

Sure! So I talked with @jmhooper and we came to the conclusion that there isn't a clear reason as to  _why_ the check was added. The front end logging is specifically for the identity verification step and in order to start that process, you already have to be authenticated.

I am more than happy to discuss this further with those who implemented this change, but as it stands, it seems to be unnecessary.

Reply:
Ok, thanks for the context 👍